tests/vnc_test.go

```diff
@@ -177,104 +165,6 @@ var _ = Describe("[rfe_id:127][crit:medium][arm64][vendor:cnv-qe@redhat.com][lev
 			_, err = wrappedRoundTripper.RoundTrip(req)
 			Expect(err).ToNot(HaveOccurred())
 		})
-
-		It("[test_id:4272]should connect to vnc with --proxy-only flag", func() {
```

Comment:
I am being lazy here, do we have unit test covering that random port is chosen?

Reply:
Unfortunately we have no unit tests for `virtctl vnc` at all. :(

Reply:
I would wait with removing the test until we can unit test it.

Reply:
Re-added the test.